src/meetings/components/sidebar/visualEffectsAccordion/VisualEffectsList.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { FC, useMemo } from 'react';
+
+import { Container, Icon, Text } from '@zextras/carbonio-design-system';
+import { map } from 'lodash';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import useVirtualBackground from '../../../../hooks/useVirtualBackground';
+import { getBackgroundImage } from '../../../../store/selectors/ActiveMeetingSelectors';
+import useStore from '../../../../store/Store';
+import { VirtualBackgroundType } from '../../../../types/store/ActiveMeetingTypes';
+
+type VisualEffectCardsProps = {
+	meetingId: string;
+};
+
+const PictureContainer = styled(Container)<{ $picture?: string | false; $isSelected: boolean }>`
+	background-size: cover;
+	background-position: center;
+	aspect-ratio: 1.2959;
+	border-radius: 0.5rem;
+	${({ $isSelected, theme }): string | false =>
+		$isSelected && `border: 1px solid ${theme.palette.success.regular};`}
+	${({ $picture, theme }): string =>
+		$picture
+			? `background-image: url(${$picture});`
+			: `background-color: ${theme.palette.gray0.regular}`};
+`;
+
+const ListContainer = styled(Container)`
+	overflow-y: scroll;
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+`;
+
+const VisualEffectsList: FC<VisualEffectCardsProps> = ({ meetingId }) => {
+	const [t] = useTranslation();
+	const setBackgroundImage = useStore((store) => store.setBackgroundImage);
+	const backgroundSelected = useStore((store) => getBackgroundImage(store, meetingId));
+
+	const { virtualBackgroundImages } = useVirtualBackground();
+
+	const backgroundTiles = useMemo(
+		() =>
+			map(VirtualBackgroundType, (element) => {
+				const isBlurOrNone =
+					element === VirtualBackgroundType.BLUR || element === VirtualBackgroundType.NONE;
+				const isSelected = element === backgroundSelected;
+
+				const changeBackground = (): void => {
+					setBackgroundImage(meetingId, element);
+				};
+
+				const elementLabel =
+					element === VirtualBackgroundType.BLUR
+						? t('meeting.visualEffects.blur', 'Blur')
+						: t('meeting.visualEffects.none', 'None');
+
+				return (
+					<Container
```

Comment:
Is a container within a container necessary? Could you use only `PictureContainer` ?